diff -ur kallithea/model/db.py kallithea_mysql/model/db.py
--- kallithea/model/db.py       2020-06-02 12:00:40.118850577 +0000
+++ kallithea_mysql/model/db.py 2020-06-02 09:05:48.990852385 +0000
@@ -2295,7 +2295,6 @@
     __tablename__ = 'user_ssh_keys'
     __table_args__ = (
         Index('usk_fingerprint_idx', 'fingerprint'),
-        UniqueConstraint('fingerprint'),
         _table_args_default_dict
     )
     __mapper_args__ = {}
